snippets/odoo-python-snippets.cson

```diff
@@ -44,15 +44,41 @@
 
 	'Many2one Field':
 		'prefix': 'oo_field_many2one'
-		'body': 'new_field_id = fields.Many2one(comodel_name="$1", string="$2", )'
+		"body": '''
+			Many2one(
+				"${10:res.partner}"${14:,
```

Comment:
I would keep `comodel_name=` and `string=`, since they are not supposed to be positional arguments: https://github.com/odoo/odoo/blob/9.0/openerp/fields.py#L1649